scripts/install_environment.sh

```diff
@@ -151,6 +151,13 @@ EOF
   sudo chmod +x $fname
 }
 
+function configure_iptables() {
```

Comment:
i think it is bad place. Try to set this inside container. I think it is bad idea to have it on each node